src/main.cpp

```diff
@@ -1,2 +1,55 @@
+#include "unix_socket_server.hpp"
+#include <iostream>
+#include <sys/un.h>
+#include <unistd.h>
 
-int main() {}
+int main() {
```

Comment:
We would prefer to have an abstraction over socket data parsing instead of having it inside the main function.

Reply:
Okay, could I crate separate file or separated function in main.cpp will be great? Its kind of entry component for the entire project, so I am not sure, what do you think ?

Reply:
> Okay, could I crate separate file or separated function in main.cpp will be great? Its kind of entry component for the entire project, so I am not sure, what do you think ?

We want to probably have separate class for that. So you will have to create header and implementation.